images/linux/scripts/installers/nodejs.sh

```diff
@@ -4,14 +4,28 @@
 ##  Desc:  Installs Node.js LTS and related tooling (Gulp, Grunt)
 ################################################################################
 
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
+
 # Install LTS Node.js and related build tools
 curl -sL https://raw.githubusercontent.com/mklement0/n-install/stable/bin/n-install | bash -s -- -ny -
 ~/n/bin/n lts
-npm install -g grunt gulp n parcel-bundler typescript newman vercel
-npm install -g --save-dev webpack webpack-cli
+# Install node modules
+node_modules=$(get_toolset_value '.node_modules[].name')
+
+# remove commenting out of this line as nmp migrated to v7
+# npm install -g $node_modules
```

Comment:
these comments can be removed, right?

Reply:
I think they should not. This command line will install all modules in time and speed up the the build a bit. We have to use the foreach loop while we live with npm6 and need a workaround for netlify